cpp/src/io/format/image_format.cpp

```diff
@@ -162,6 +162,13 @@ ImageMetadata& ImageMetadata::level_downsamples(const std::pmr::vector<float>& l
     return *this;
 }
 
+ImageMetadata& ImageMetadata::level_tile_sizes(const std::pmr::vector<uint32_t>& level_tile_sizes)
+{
+    level_tile_sizes_ = std::move(level_tile_sizes);
```

Comment:
Can we use `std::move` on `const &`? Or should the input be `&&`?

Reply:
Thanks a lot @jakirkham for pointing out that !!!
Without your review, it would end up with inefficient code (copy instead of move).

Thank you so much!